lib/crystalball/map_generator/factory_bot_strategy/factory_runner_patch.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Crystalball
+  class MapGenerator
+    class FactoryBotStrategy
+      # Module to add new patched `run` method to FactoryBot::FactoryRunner
+      module FactoryRunnerPatch
+        class << self
+          # Patches `FactoryBot::FactoryRunner#run`. Renames original `run` to `cb_original_run` and
+          # replaces it with custom one
+          def apply!
+            FactoryBotStrategy.factory_bot_constant::FactoryRunner.class_eval do
```

Comment:
This way to patch is not compatible if someone uses `Module.prepend` and tries to call an original method with `super` - it will create an endless cycle
I have an idea to use `Module.prepend` here and for all our patches to handle such case
pros: `super` works for everyone 
cons: we can't revert our patches anymore. But do we need to revert them at all?
WDYT @pluff @jaimerson ?

Reply:
I think reverting the patches is necessary to not have unintended consequences on the next strategies. Do we have this issue with `prepend` right now? If not I think we can keep it this way.

Reply:
We have it on another project. Exactly the case I described